pkg/controllers/clusterregistration/controller.go

```diff
@@ -169,7 +169,7 @@ func (h *handler) OnSecretChange(key string, secret *v1.Secret) (*v1.Secret, err
 		return secret, nil
 	}
 
-	if time.Now().Sub(secret.CreationTimestamp.Time) > deleteSecretAfter {
+	if time.Since(secret.CreationTimestamp.Time) > deleteSecretAfter {
```

Comment:
What is the linter error here? Does it provide any error code?

Reply:
You can run the linter locally like this:
`golangci-lint run --config=.golangci-full.json`

The change resulted from the gosimple linter: 

```
pkg/controllers/clusterregistration/controller.go:172:5: S1012: should use `time.Since` instead of `time.Now().Sub` (gosimple)
	if time.Now().Sub(secret.CreationTimestamp.Time) > deleteSecretAfter {
```